analytic_engine/src/instance/write_worker.rs

```diff
@@ -956,7 +956,12 @@ impl WriteWorker {
             waiter,
         };
 
-        self.instance.schedule_table_compaction(request).await;
+        let table = request.table_data.name.clone();
+        let succeed = self.instance.schedule_table_compaction(request).await;
+        if !succeed {
+            error!("Failed to schedule compaction, table:{table}");
```

Comment:
I think debug log is enough. otherwise it will log too much in write-heavy case.

Reply:
Here is triggered by manual compaction, I guess the message will seldom occur.